### DISCUSSION TITLE
How to disable the automatic collapse of Signal items?

### DISCUSSION THREAD

**Discussion by generateui** [datalust/seq-tickets#1918] [Help]
In the Signals pane we have about 80+ signals. However, most of them are hidden while a button is shown `+ 68 more`. When clicking, the list of signals expands with all the items. However, the behavior is that it autocollapses fairly quickly and selection of a signal.

This collapsing plus its behavior makes me click **a lot**. Furthermore, it seems completely unecessary to collapse these items. It also reduces the ability to easily get an overview.

How to remove this button and its behavior and just always show all items in that list instead? That would make my life a lot better. Thanks in advance!

**Comment by nblumhardt** [comment]
Hi @generateui,

You can do this by de-selecting the "Personal" workspace you likely have selected in the top left of the nav bar:

![image](https://github.com/datalust/seq-tickets/assets/342712/092b67bd-e3fb-4ff7-9cd1-f285c4295ccc)

Choosing "None" in the workspace list will close the workspace and you'll see everything on the server by default.

Hope this helps,
Nick

**Comment by MoazAlkharfan** [comment]
I just have to say that this behavior makes no sense at all, and it's very annoying for the signals to automatically fold with no control over the behavior.
